subsys/partition_manager/pm.yml.vpr_launcher

```diff
@@ -0,0 +1,7 @@
+vpr_launcher:
```

Comment:
Should we have kconfig options for these values so that they can be changed from kconfig instead of directly modifying these files?

Reply:
Currently they must match DT.

Reply:
Added Kconfig that takes value from DT.